File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/SensitiveWordsVO.kt
package com.sdkwork.app

data class SensitiveWordsVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val hasSensitiveWords: Boolean? = null,
    val count: Int? = null,
    val words: List<WordHit>? = null,
    val processedText: String? = null,
    val riskLevel: String? = null
)

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/sensitive_words_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class SensitiveWordsVO:
    """敏感词检测响应"""
    created_at: str = None
    updated_at: str = None
    has_sensitive_words: bool = None
    count: int = None
    words: List[WordHit] = None
    processed_text: str = None
    risk_level: str = None
